exercises/saddle-points/example.rs

```diff
@@ -0,0 +1,23 @@
+
+pub fn find_saddle_points(input: Vec<Vec<u64>>) -> Vec<(u64, u64)>{
+    let mut saddle_points: Vec<(u64, u64)> = Vec::new();
+
+    let row_index = input.len();
+    let column_index = input[0].len();
+
+    for i in 0..row_index {
+        for j in 0..column_index {
+            let row = &input[i];
+            let column = input.iter().map(|x| x[j]).collect::<Vec<u64>>();
+
+            let max = row.iter().max().unwrap();
```

Comment:
since this is only based on `row`, which is only based on `i` and not `j`, it would be possible to assign it in the outer loop and not the inner one. Unless that makes the code less understandable.

Reply:
If the matrix is empty the two for loops prevent an unwrap on  ```row``` in line 13.